app/src/main/java/net/squanchy/navigation/RoutingInjector.java

```diff
@@ -21,6 +23,8 @@ public static RoutingComponent obtain(Activity activity) {
                 .navigationModule(new NavigationModule())
                 .signInModule(new SignInModule())
                 .routingModule(new RoutingModule())
+                .proximityFeatureModule(new ProximityFeatureModule())
```

Comment:
Used by the `Onboarding` class (through the `OnboardingModule`) to decide whether we can show the Proximity onboarding page (https://github.com/rock3r/squanchy/pull/256/files#diff-e08306406758fca42ffe92c82be63267R35)